packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -28,92 +28,66 @@ import {
      LocalReferenceCollection,
      LocalReferencePosition,
 } from "./localReference";
+import {
+	BaseSegment,
```

Comment:
no real changes here. just moves between this file and mergeTreeNodes to minimize export surface